c_glib/arrow-glib/input-stream.h

```diff
@@ -66,6 +66,8 @@ GArrowBuffer *garrow_seekable_input_stream_read_at(GArrowSeekableInputStream *in
                                                    gint64 position,
                                                    gint64 n_bytes,
                                                    GError **error);
+const gchar *garrow_seekable_input_stream_peek(GArrowSeekableInputStream *input_stream,
```

Comment:
Could you add `GARROW_AVAILABLE_IN_0_12`?

Reply:
I've added it.